clang/lib/Sema/TreeTransform.h

```diff
@@ -15661,12 +15661,16 @@ TreeTransform<Derived>::TransformCXXFoldExpr(CXXFoldExpr *E) {
       return true;
   }
 
+  if (*NumExpansions == 1) {
```

Comment:
Should we set the flag unconditionally (regardless of the number of expansions)?